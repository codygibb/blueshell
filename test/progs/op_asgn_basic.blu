x := 3
x += 4
print x
x -= 5
print x
x *= 7
print x
x /= 4
print x
