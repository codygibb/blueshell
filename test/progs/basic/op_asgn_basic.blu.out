7
2
14
3
